### PR TITLE
Introduce release script to automate release and image publish to docker hub

### PR DESCRIPTION
**Pull request by kabirbaidhya**
Introduce release script to automate release and image publish to docker hub.
 - Publish new release to docker hub (pushes both `latest` and new version tags)
 - Publish new release to GitHub

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+set -e
+
+# Download semver
+sudo curl https://raw.githubusercontent.com/fsaintjacques/semver-tool/2.1.0/src/semver --silent -o /usr/local/bin/semver && sudo chmod +x /usr/local/bin/semver
+
+last_tag=$(git tag --sort=-creatordate | head -n 1)
+new_tag=$(semver bump patch "$last_tag")
+timestamp=$(date -u +%Y%m%d%H%M%S)
+new_version=$(if [ "$BRANCH" == "master" ]; then echo "${new_tag}"; else echo "${new_tag}-${BRANCH}.$timestamp"; fi)
```
Comment (mesaugat) [inline release.sh]: `==` is pattern matching.

Use `=`.

**Comment by mesaugat** [comment]
Pushed to Docker Hub. ¯\\_(ツ)_/¯

**Comment by kabirbaidhya** [comment]
@mesaugat This happened - how come? 
image
Travis not allowed to push to GitHub?

**Comment by mesaugat** [comment]
@kabirbaidhya We'll need to add `laudio-bot`'s SSH key. Wait I'll add it.

**Comment by mesaugat** [comment]
And @travis-ci only supports SSH key for private repositories. Haha. #legacy

Now we will need to encrypt and commit the SSH key in the repository and decrypt it during build time.

**Comment by kabirbaidhya** [comment]
Wow :1st_place_medal:

**Comment by kabirbaidhya** [comment]
@mesaugat What if we want to add `hub` here? Will it work with token or ssh?

**Comment by kabirbaidhya** [comment]
Brother, this is a public repository so let's not put JIRA tickets here :smile:

**Comment by mesaugat** [comment]
I forgot. 🗡

**Comment by mesaugat** [comment]
@kabirbaidhya It will work with a GITHUB_TOKEN and hub. That way we can release it too.

**Comment by kabirbaidhya** [comment]
Great. Are we using it for `laudio/docker` too?

**Review by mesaugat — APPROVED** [review]

**Review by mesaugat — APPROVED** [review]
LGTM! 🥇

**Pull request merged**